kikar_hamedina/core/urls.py

```diff
@@ -118,4 +119,4 @@
                            name='django.contrib.sitemaps.views.sitemap'),
                        ) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
-urlpatterns = format_suffix_patterns(urlpatterns)
+urlpatterns = format_suffix_patterns(urlpatterns)
```

Comment:
This is a mistake. it is not relevant. pls do not update the main repository with this file.